services/s3/src/main/java/software/amazon/awssdk/services/s3/S3CrtAsyncClientBuilder.java

```diff
@@ -322,11 +322,13 @@ default S3CrtAsyncClientBuilder retryConfiguration(Consumer<S3CrtRetryConfigurat
      *     <li>You want more fine-grained control over the {@link ThreadPoolExecutor} used, such as configuring the pool size
      *     or sharing a single pool between multiple clients.
      *     <li>You want to add instrumentation (i.e., metrics) around how the {@link Executor} is used.
-     *     <li>You know, for certain, that all of your {@link CompletableFuture} usage is strictly non-blocking, and you wish to
-     *     remove the minor overhead incurred by using a separate thread. In this case, you can use
-     *     {@code Runnable::run} to execute the future-completion directly from within the I/O thread.
      * </ol>
      *
```

Comment:
nit: can we add `<p>` tag? https://github.com/aws/aws-sdk-java-v2/blob/master/docs/design/APIReference.md